src/components/Dialog/Dialog.ts

```diff
@@ -352,6 +323,24 @@ const showWriteErrorDialog = ({
   }
 };
 
+export const notifyResult = (
```

Comment:
これ場合によってはnotifyしないので、ドキュメントでそれを書いてあげるとちょっとだけ優しいかもと思いました！
```suggestion
/** 保存結果に応じてユーザーに通知する。キャンセルされた場合は何もしない。 */
export const notifyResult = (
```